garak/generators/huggingface.py

```diff
@@ -305,6 +306,7 @@ def _call_model(self, prompt: str, generations_this_call: int = 1) -> List[str]:
             headers=self.headers,
             json=payload,
             timeout=(20, 90),  # (connect, read)
+            proxies={'http': 'http://127.0.0.1:7890/', 'https': 'http://127.0.0.1:7890/'}
```

Comment:
```suggestion
```

Reply:
Yes, those are just prepare for local version. (I cannot do development without those proxy settings🤣)